react-app/package.json

```diff
@@ -10,24 +10,26 @@
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
     "i": "^0.3.6",
-    "node-sass": "^4.11.0",
     "npm": "^6.9.0",
     "react": "^16.5.2",
     "react-dom": "^16.5.2",
     "react-fast-compare": "^2.0.4",
     "react-router": "^5.0.0",
     "react-router-dom": "^5.0.0",
-    "react-scripts": "^2.1.8",
-    "react-styleguidist": "^9.0.6"
+    "react-scripts": "^3.2.0",
+    "react-styleguidist": "^9.0.6",
+    "register-service-worker": "^1.6.2",
+    "sass": "^1.23.7"
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build && clientlib --verbose",
+    "build": "react-scripts build && node ./utils/service-worker/generate.js && clientlib --verbose",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "styleguide": "styleguidist server",
     "styleguide:build": "styleguidist build"
   },
+  "homepage": "/etc.clientlibs/wknd-events/clientlibs/react-app/resources",
```

Comment:
@samuelmeuli -> this seems a bit strange..is it just to facilitate relative path resolution?

Reply:
This path is prepended to the files in `asset-manifest.json` and `precache-manifest.json`, i.e. it's required for code splitting and caching to work. But maybe it's less confusing if the `PUBLIC_URL` env variable is used instead of the `homepage` key

Reply:
I've changed this and documented why it's required

Reply:
nice, i like it. thanks @samuelmeuli!